src/server/honeypot.ts

```diff
@@ -73,7 +73,10 @@ export class Honeypot {
   }
 
   protected get validFromFieldName() {
-    return this.config.validFromFieldName ?? DEFAULT_VALID_FROM_FIELD_NAME;
+    if (this.config.validFromFieldName === undefined) {
+      return DEFAULT_VALID_FROM_FIELD_NAME;
+    }
+    return this.config.validFromFieldName;
```

Comment:
Now we only default if the value is undefined. Otherwise we return it as-is so `null` can be returned.